src/object_properties.h

```diff
@@ -66,6 +74,10 @@ struct ObjectProperties
 	bool use_texture_alpha = false;
 	bool shaded = true;
 	bool show_on_minimap = false;
+	u16 engine_mask = 0;
+	float drowning_interval = 2.0f;
```

Comment:
4 new properties for every object for this seems somewhat wasteful. Shouldn't this apply just to players? Modders can implement their own logic for luaentities anyways.